evaluation/tasks/mrpc-confirmation/mrpc-confirmation.py

```diff
@@ -0,0 +1,152 @@
+from datasets import load_dataset
+from jinja2 import Template
+from torch.utils.data import Dataset
+from tqdm import tqdm
+import torch
+
+from evaluation.tasks.auto_task import AutoTask
+from evaluation.utils.log import get_logger
+
+TEMPLATE_PARAPHRASE = Template(
+    """Paraphrase the following sentence: {{sent1}}
+    """
+)
+
+TEMPLATE_CONFIRMATION = Template(
+    """Sentence 1: {{sent1}}
+Sentence 2: {{sent2}}
+Do these two sentences convey the same meaning? Yes or no?
+    """
+)
+
+
+class MRPCDataset(Dataset):
+    def __init__(self, tokenizer):
+        super().__init__()
+        mrpc = load_dataset("glue", "mrpc", split="validation")
+        self.items = []
+
+        for sample in mrpc:
+            # detokenize the text, since MRPC is tokenized
+            # with MosesDetokenizer('en') as detokenize:
+            #    sample["sentence1"] = detokenize(sample["sentence1"].split())
+            # print(sample["sentence1"])
+            prompt = TEMPLATE_PARAPHRASE.render(
+                sent1=sample["sentence1"],
+            )
+
+            # Tokenize and construct this sample
+            inputs = tokenizer(
+                prompt,
+                return_tensors="pt",
+                truncation=True,
+            )
+            self.items.append(
+                {
+                    "prompt": prompt,
+                    "sentence1": sample["sentence1"],
+                    "input_ids": inputs["input_ids"],
+                    "attention_mask": inputs["attention_mask"],
+                    "input_len": inputs["attention_mask"].shape[1],
+                }
+            )
+
+    def __len__(self):
+        return len(self.items)
+
+    def __getitem__(self, index):
+        return self.items[index]
+
+
+class MRPCNegativeTask(AutoTask):
```

Comment:
rename to match file
```suggestion
class MRPCConfirmationTask(AutoTask):
```